doc/source/mast_data_products/hap-parameters.rst

```diff
@@ -502,4 +538,4 @@ MAS_TO_ARCSEC: float (*default=1000*)
     Conversion factor from milliarcseconds to arcseconds.
 
 GOOD_FIT_QUALITY_VALUES: int (*default=[1, 2, 3, 4]*)
-    The fit_quality (see above) flag values that are allowable for a successful fit. 
\ No newline at end of file
+    The fit_quality (see above) flag values that are allowable for a successful fit.
```

Comment:
I think you should add section headings where the discussions of the catalog configuration file and the quality control (of the catalog information) configuration file will eventually be written.

Reply:
I created [a ticket](https://jira.stsci.edu/browse/HLA-1352) for working on those. I tentatively assign you, as you know more about the catalogs, but I can also add empty sections in this PR.